mysql-test/main/system_mysql_db_fix40123.result

```diff
@@ -108,6 +108,7 @@ user	CREATE TABLE `user` (
   `plugin` char(64) CHARACTER SET latin1 NOT NULL DEFAULT '',
   `authentication_string` text COLLATE utf8_bin NOT NULL,
   `password_expired` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N',
+  `account_locked` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N',
```

Comment:
Is that so? I don't see what changes in this commit could explain this difference in results